### PR TITLE
QDB-3131 - Symbols python API

### PR DESCRIPTION
**Pull request by Dwarfobserver**
Update Python API to with symbol API.

### CODE REVIEW

File: tests/test_table.py [hunk 1]
```diff
@@ -539,3 +557,72 @@ def test_string_get_ranges(table, intervals):
             _string_col_name(table),
             inserted_string_data[0],
             inserted_string_data[1])
+
+###
+# Symbol tests
```
Comment (mkurdej) [inline tests/test_table.py:562]: It's time to split this file...

File: tests/test_batch_inserter.py [hunk 2]
```diff
@@ -221,11 +238,12 @@ def test_push_truncate_implicit_range(qdbd_connection, table, many_intervals):
 
     # Generate our dataset
     data = _generate_data(len(many_intervals))
-    # (doubles, integers, blobs, strings, timestamps) = data
-    (doubles, _, _, _, _) = data
+    # (doubles, integers, blobs, strings, timestamps, symbols) = data
+    (doubles, _, _, _, _, syms) = data
```
Comment (mkurdej) [inline tests/test_batch_inserter.py:242]: Please one name everywhere... `symbols`.

File: tests/test_batch_inserter.py [hunk 3]
```diff
@@ -221,11 +238,12 @@ def test_push_truncate_implicit_range(qdbd_connection, table, many_intervals):
...
 
     # Insert once
     inserter = qdbd_connection.inserter(_make_inserter_info(table))
+    print("SYMBOLS 1: {}".format(syms))
```
Comment (mkurdej) [inline tests/test_batch_inserter.py:246]: Debug print to remove? (Here and below)

File: tests/test_batch_inserter.py [hunk 4]
```diff
@@ -17,14 +17,17 @@ def _row_insertion_method(
...
+        timestamps,
+        symbols):
     for i in range(len(dates)):
         inserter.start_row(dates[i])
         inserter.set_double(0, doubles[i])
         inserter.set_blob(1, blobs[i])
         inserter.set_string(2, strings[i])
         inserter.set_int64(3, integers[i])
         inserter.set_timestamp(4, timestamps[i])
+        print("@ insert {}: {}".format(type(symbols[i]), symbols[i]))
```
Comment (mkurdej) [inline tests/test_batch_inserter.py:29]: Debug?

File: tests/test_batch_inserter.py (hunk 3 above)
Comment (Dwarfobserver) [reply tests/test_batch_inserter.py:246]: Oops indeed

File: tests/test_table.py (hunk 1 above)
Comment (Dwarfobserver) [reply tests/test_table.py:562]: 😢

Comment (mkurdej) [reply tests/test_table.py]: Nice!

**Review by mkurdej — APPROVED** [review]
Nice! LGTM if CI tests pass 👍

**Pull request merged**